crosswalks-sssom/README.md

```diff
@@ -24,28 +28,47 @@ The methodology for generating the files required to create the crosswalk is as
-       - close_match: The subject and the object are sufficiently similar that they can be used interchangeably in some information retrieval applications. We do NOT recommend using this.
-   - combined_mapping: It applies to a part-of relationship.
-     This column defines the formula used to combine terms between schemes.
-     For example, if the source vocabulary has the term date, and the target vocabulary has the terms year and month, the CSV should contain two separate lines.
-     This column will contain the formula date: `$(year) + $(month)`.
-     ** This is a strategy NOT supported in SSSOM. **
-
-   **Considerations**
-
-   1. Hierarchical structures.
-      Mapping of terms can follow this structure
-      vocab.owner.name = codemeta.author.name
-      In these cases, you will need to include the parent term, excluding the reference to the semantic artifact, as it is already included in the YAML file. Hierarchy will be delimited by the character '/'. For example:
+       - close_match: The subject and the object are sufficiently similar that they can be used interchangeably in some information retrieval applications. For example, if the source and target properties refer to dates, but the format is different.
+   - combined_mapping: **It applies ONLY to a part-of relationship**.
+     This column defines the formula used to combine terms between schemas.
+     For example, if the source vocabulary has the term `date`, and the target vocabulary has the terms `year` and `month`, the combined_mapping should be as follows: $(year) + $(month)`.
+   - comments: additional (optional) annotations about the mapping itself.
+
+5. **Generate the SSSOM File**<br>
+   **Note:** This script uses the library sssom. Please, install it before executing the script.
+   ```bash
+   pip install sssom
+   ```
+   Execute the script (generate_crosswalk.py) to generate an .sssom.tsv file from the CSV.
+   ```bash
+   python ./generate_crosswalk.py examples/bibtex-codemeta-mappings.yaml examples/bibtex-codemeta-mappings.csv
+   ```
+### FAQ
+Below we have collected frequently asked questions 
+
+#### How do I mint a persistent identifier for my mapping/crosswalk?
+Create a folder with a unique name. The folder name will be used to create a unique identifier. No need to anything else!
+The corresponding identifier will be `https://w3id.org/codemeta/mappings/{YOUR_ID}`, where `{YOUR_ID}` is the id of your mapping.
+
+For example, for the `bibtex` mapping, the corresponding identifier is [https://w3id.org/codemeta/mappings/bibtex](https://w3id.org/codemeta/mappings/bibtex).
+#### How do I fill in the crosswalk table? What does "more specific" and "more general" means?
+The crosswalks indicate term relationships between two schemas. The `source_term` is the term from CodeMeta`. The `target_term` is is the term from the mapped schema.
```

Comment:
Isn't CodeMeta the target?
We translate a collection of vocabularies to CodeMeta?